test-configs.yaml

```diff
@@ -161,9 +161,12 @@ test_plans:
         panfrost_submit
 
   kselftest:
-    rootfs: buildroot_kselftest_ramdisk
+    rootfs: debian_buster_nfs
+    pattern: 'kselftest/{category}-{method}-{protocol}-nfs-kselftest-template.jinja2'
     filters:
       - passlist: {defconfig: ['kselftest']}
+    params:
+      job_timeout: '60'
```

Comment:
This reminds me, we should change the parameter name to make it clear the timeout is a number of minutes...